spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationInterpreter_ArraySuite.scala

```diff
@@ -0,0 +1,218 @@
+        |+---------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        ||[,,]                                                           |[[stdCastError, E00000, Standardization Error - Type cast, arrayField[*], [00:00:00 01.12.2018], []], [stdCastError, E00000, Standardization Error - Type cast, arrayField[*], [00:10:00 02.12.2018], []], [stdCastError, E00000, Standardization Error - Type cast, arrayField[*], [00:20:00 03.12.2018], []]]|
+        ||[,,]                                                           |[[stdCastError, E00000, Standardization Error - Type cast, arrayField[*], [00:00:00 01.12.2019], []], [stdCastError, E00000, Standardization Error - Type cast, arrayField[*], [00:10:00 02.12.2019], []], [stdCastError, E00000, Standardization Error - Type cast, arrayField[*], [00:20:00 03.12.2019], []]]|
+        ||[2020-01-12 00:00:00, 2020-12-02 00:10:00, 2020-12-03 00:20:00]|[]                                                                                                                                                                                                                                                                                                             |
+        |+---------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+        |
+        |""".stripMargin.replace("\r\n", "\n")
+    val expectedSchema =
+      "root\n"+
+      " |-- arrayField: array (nullable = true)\n" +
+      " |    |-- element: timestamp (containsNull = true)" +
+      errColSchema
+
+    val std = StandardizationInterpreter.standardize(src, desiredSchema, "").cache()
+    assert(std.schema.treeString == expectedSchema)
+    assert(std.dataAsString(false) == expectedData)
+  }
+
+  test("Array of timestamps with pattern defined") {
+    val seq  = Seq(
+      Array("00:00:00 01.12.2018", "00:10:00 02.12.2018","00:20:00 03.12.2018"),
+      Array("00:00:00 01.12.2019", "00:10:00 02.12.2019","00:20:00 03.12.2019"),
+      Array("2020-01-12 00:00:00" ,"2020-12-02 00:10:00","2020-12-03 00:20:00")
+    )
+    val src = seq.toDF(fieldName)
+    val desiredSchema = generateDesiredSchema(TimestampType, s""""${MetadataKeys.Pattern}": "HH:mm:ss dd.MM.yyyy"""")
+
+    val expectedData =
```

Comment:
When switching from Spark 2.2.x to 2.4.4 we have noticed that the format of `.show()` is changed. So in order to make unit tests lee fragile we prefer exporting expected data as JSONs in resources.
Dataframe JSONs can be prettified for easier comparison between the expected and actual data.
This method can be used for that: `JsonUtils.prettyJSON(inputJson)`.

But it is just a suggestion - I'm fine with this implementation as well. The switch to the different way could be done later in bulk.